client/src/App.js

```diff
@@ -1,8 +1,34 @@
-import './App.css';
+//import 
+import { Route, Switch } from 'react-router-dom'
+import './styles/App.css';
+
+import Home from './components/Home'
+import NavBar from './components/NavBar'
+import EditDelete from './components/EditDelete'
+import Facts from './components/Facts'
+import NotFound from './components/404'
 
 function App() {
   return (
-    <h1>Today I Learned:</h1>
+
+    <div id='main-container'>
+
+
+      <NavBar />
+      <Switch>
+        {/*Route directing the user to Home Page */}
+        <Route exact path='/' component={Home} />
+        {/*Route directing the user to Facts Page */}
+        <Route path="/facts" component={Facts} />
+        {/*Route directing the user to specific entry page Page */}
+        <Route path="/facts/:objectId" component={EditDelete} />
```

Comment:
Because of the way that Routes work, Your second route is never being triggered. You need to put the more specific one at the top. Currently your `path="/facts"` is catching everything and not continuing through to your `EditDelete` component